### PR TITLE
fix dm openapi http method

### LINKED ISSUE #8188: dm openapi 需要把PATCH 修改为POST
## Error 报告

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the document directory for the problem?

dm/dm-open-api.md

2. How would you like to improve it?
dm openapi 需要把所有的PATCH 修改为POST

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/dm/pull/2187

- This PR is translated from:
- Other reference link(s): close #8188 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-open-api.md [hunk 1]
```diff
@@ -392,14 +392,14 @@ curl -X 'PATCH' \
 
 ### 请求 URI
 
- `PATCH /api/v1/sources/{source-name}/stop-relay`
+ `POST /api/v1/sources/{source-name}/stop-relay`
```
Comment (lance6716) [inline dm/dm-open-api.md:395]: https://github.com/pingcap/tiflow/blob/3ab3c7d2248193ae356a00a7a720f4eecf7ad709/dm/dm/master/openapi.go#L277

can you provide a reference file that review can check the description?

Comment (Ehco1996) [reply dm/dm-open-api.md:395]: thanks, the source-truth is in [spec yaml](https://github.com/pingcap/tiflow/blob/17e48b2c64b9d3dfc0cbd210c3ca0e882c437abb/dm/openapi/spec/dm.yaml#L180), the above comment was outdated, i will fix those comments in other pr

**Review by lance6716 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Comment by TomShawn** [comment]
@Ehco1996 @lance6716 Can I merge this PR？

**Comment by Ehco1996** [comment]
> @Ehco1996 @lance6716 Can I merge this PR？

yes please

**Pull request merged**